frontend/src/components/form/InputText.vue

```diff
@@ -11,31 +11,41 @@ type Props = {
   name: string;
   placeholder?: string;
   disabled?: boolean;
+  bold?: boolean;
 };
 
 const props = withDefaults(defineProps<Props>(), {
   helpText: '',
   type: 'text',
   label: '',
   placeholder: '',
-  disabled: false
+  disabled: false,
+  bold: true
 });
 
 const { errorMessage, value } = useField<string>(toRef(props, 'name'));
 </script>
 
 <template>
   <div class="field col">
-    <label :for="name">{{ label }}</label>
+    <label
+      :class="{ 'font-bold': bold }"
+      :for="name"
+    >
+      {{ label }}
+    </label>
     <InputText
       v-model.trim="value"
       :aria-describedby="`${name}-help`"
       :name="name"
       :placeholder="placeholder"
-      :class="'w-full ' + { 'p-invalid': errorMessage }"
+      class="w-full"
+      :class="{ 'p-invalid': errorMessage }"
       :disabled="disabled"
     />
     <small :id="`${name}-help`">{{ helpText }}</small>
-    <ErrorMessage :name="name" />
+    <div>
+      <ErrorMessage :name="name" />
+    </div>
```

Comment:
Is the new div wrapper addressing some kind of strange spacing/layout issue? Usually we try to avoid nesting more DOM elements whenever possible.